lapis2-docs/package.json

```diff
@@ -10,7 +10,8 @@
         "astro": "astro",
         "check-format": "prettier --check \"**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\"",
         "format": "prettier --write \"**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\"",
-        "check-types": "tsc --noEmit && CONFIG_FILE=../siloLapisTests/testData/testDatabaseConfig.yaml astro check"
```

Comment:
I don't know why, but `astro check` wouldn't be successful for weird reasons. I extracted #534. Until then, we need `astro sync` to generate the `astro:content` module for typescript.